.github/workflows/readme-check.yml

```diff
@@ -0,0 +1,22 @@
+name: Check README Updates
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+jobs:
+  check-readme:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+
+      - name: Make check-missing-readme-generation.sh executable
+        run: chmod +x .github/scripts/check-missing-readme-generation.sh
```

Comment:
You can delete this step and instead run `chmod +x .github/scripts/check-missing-readme-generation.sh` locally, commit and push the change.